src/app/components/client/LocationAutocomplete.module.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../tokens";
+
+.locationItem {
+  font: $text-body-sm;
+  padding: $spacing-sm calc($spacing-sm * 1.5);
```

Comment:
suggestion (nit): Add comment about calculated number

Reply:
Added in [28c818f](https://github.com/mozilla/blurts-server/pull/3217/commits/28c818f7de9d91585f047dd5a89cf478a30d9909).